components/esp_http_server/include/esp_http_server.h

```diff
@@ -39,7 +39,7 @@ extern "C" {
         .lru_purge_enable   = false,                    \
         .recv_wait_timeout  = 5,                        \
         .send_wait_timeout  = 5,                        \
-};
+}
```

Comment:
Would the global context and function pointers need to be set to NULL under HTTPD_DEFAULT_CONFIG?

Reply:
not needed, when you initialize a struct with {}, the unspecified members are set to zero automatically

Reply:
Please note though that if this header is used from a C++ source file, compiler will complain about the missing initializers, or if the order of initializers does not match the order of structure fields. So it is still desirable to initialize all the fields.

Reply:
initializers added